client/client.go

```diff
@@ -840,6 +840,11 @@ type (
 		// NOTE: Experimental
 		UpdateWorkflow(ctx context.Context, options UpdateWorkflowOptions) (WorkflowUpdateHandle, error)
 
+		// UpdateWithStartWorkflow intercepts client.Client.UpdateWithStartWorkflow.
+		//
+		// NOTE: Experimental
+		UpdateWithStartWorkflow(ctx context.Context, options UpdateWorkflowOptions, startOperation WithStartWorkflowOperation) (WorkflowUpdateHandle, error)
```

Comment:
```suggestion
		UpdateWithStartWorkflow(ctx context.Context, options UpdateWithStartWorkflowOptions) (WorkflowUpdateHandle, error)
```

I think an options structure here accepting the two possible parameters here is more future proof

Reply:
Thanks, discussed internally and taken this suggestion: 88e4896b3ae8b035af6c9c481601660ae6502134